eternalcore-core/src/main/java/com/eternalcode/core/configuration/implementation/PluginConfiguration.java

```diff
@@ -216,7 +216,7 @@ public static class Items extends OkaeriConfig {
     @Comment("")
     @Comment("# Server Links Configuration")
     @Comment("# Settings for server link management")
-    ServerLinksConfig serverLinks = new ServerLinksConfig();
+    public ServerLinksConfig serverLinks = new ServerLinksConfig();
```

Comment:
According to the new standard, we should avoid direct references to `PluginConfiguration`. Keeping config section instances package-private helps prevent accidental usage. Instead, we should rely on interfaces and their implementations to access configuration data safely and consistently. Please remove `public` keyword, and why you added this?